LCU.NET.Extras/Game/Detector.cs

```diff
@@ -15,12 +15,14 @@ public abstract class Detector
     {
         protected Container<Actuator.State> State { get; private set; }
         protected ILoL LoL { get; }
+        protected IUiActuator MainWindow { get; }
```

Comment:
I don't really like the naming. I'ts like that to keep the original names, but is there a better way?